src/v/model/record_batch_types.h

```diff
@@ -43,7 +43,8 @@ enum class record_batch_type : int8_t {
     version_fence = 23,              // version fence/epoch
     tx_tm_hosted_trasactions = 24,   // tx_tm_hosted_trasactions_batch_type
     prefix_truncate = 25,            // log prefix truncation type
-    MAX = prefix_truncate
+    tx_registry = 26,                // tx_registry_batch_type
```

Comment:
i am wondering if we need a special batch type for this ? the only batch types that are in the tx_registry partition are of that type

Reply:
I was thinking of having tx-registry's batch sub types encoded as a part of tx_registry batch see tx_registry_stm::batch_subtype::tx_mapping